.travis.yml

```diff
@@ -73,12 +75,21 @@ install:
     sudo snap install snapcraft --classic
     $MAGE jsSDK:deps js:deps
   fi
+- |
+  if [[ "$RUNTYPE" == "doc" ]]; then
+    set -e
+    sudo apt-get -y update
+    sudo apt-get install -y snapd
+    sudo snap install hugo
+    $MAGE docs:deps
```

Comment:
@rvolosatovs can you test this?

Reply:
Isn't it already tested by travis ?

Reply:
I think we should use go module system to run(versioned) `hugo`.